### ISSUE TITLE
Question About Training Details

### ISSUE THREAD

**Issue by kongwanbianjinyu** [icandle/CAMixerSR#24]
Hello author,

I want to reproduce the result of lightweight SR x4 task and I try to follow the training process using 2GPUs with batch size = 16 for each one. The window_size is 16x16 and feature_dim is 60.

- My environment version information is: 
```
BasicSR: 1.4.2
PyTorch: 2.1.1+cu121
TorchVision: 0.16.1+cu121
```
- Here is my training config file: [train_example.txt.txt](https://github.com/user-attachments/files/15962215/train_example.txt.txt)

- Due to the error of can't find "CAMixerSR" architecture, I modify the train.py as

```
import os.path as osp
import archs  
import models  
from basicsr.train import train_pipeline
if __name__ == '__main__':
    root_path = osp.abspath(osp.join(__file__, osp.pardir))
    train_pipeline(root_path)
```

- The training script is:
```
CUDA_VISIBLE_DEVICES=0,1 torchrun --nproc_per_node=2 --master_port 29500 basicsr/train.py \
 -opt /home/jiachen/CAMixerSR/codes/options/train/train_example.yml \
 --launcher pytorch \
```

I training for 500k iters and the best result is get at 440k iters:
![result](https://github.com/icandle/CAMixerSR/assets/39997683/17b925ce-a3a5-402e-b3aa-9148976fa74b)

The performance is not as good as the pretrained model. Is there anything that I missed? Any kinds of help is appreciated. I'm looking forward to your reply!

Thanks,
jiachen

**Comment by icandle** [comment]
Thanks for your attention to our work. Due to the batch size 16x2 being smaller than our settings 16x4, the training period should be extended and the learning rate needs to adjust to achieve the similar performance. For simplicity, you may use the latest model for a quick finetuning with the same confifure, which would improve the performance.
